mpp/shmem.h.in

```diff
@@ -17,9 +17,18 @@
 #define SHMEM_H
 
 #include <stddef.h>
-#include <complex.h>
+
+#ifdef __cplusplus
+#  define shmem_complex_t _Complex
```

Comment:
I do not understand this.  `_Complex` is not part of ISO C++, at least not that I can see.